docs/learning/tutorial.soy

```diff
@@ -41,6 +41,42 @@
 </blockquote>
 </p>
 
+<div class="toggler">
+  <table>
+    <tr>
+      <td><span><strong>Platform:</strong></span></td>
+      <td>
+        <a href="javascript:void(0);" class="button-android" onclick="display('platform', 'android')">Android</a>
+        <a href="javascript:void(0);" class="button-ios" onclick="display('platform', 'ios')">iOS</a>
```

Comment:
Let's not include this since we don't support it

Reply:
Fixed